third_party/libusb/webport/src/libusb_js_proxy.cc

```diff
@@ -1175,4 +1097,47 @@ int LibusbJsProxy::LibusbHandleEventsWithTimeout(libusb_context* context,
   return LIBUSB_SUCCESS;
 }
 
+int LibusbJsProxy::DoGenericSyncTranfer(libusb_transfer_type transfer_type,
+                                        libusb_device_handle* device_handle,
+                                        unsigned char endpoint_address,
+                                        unsigned char* data,
+                                        int length,
+                                        int* actual_length,
+                                        unsigned timeout) {
+  GOOGLE_SMART_CARD_CHECK(transfer_type == LIBUSB_TRANSFER_TYPE_BULK ||
+                          transfer_type == LIBUSB_TRANSFER_TYPE_INTERRUPT);
+
+  // Implement the synchronous transfer in terms of asynchronous one.
+  libusb_transfer transfer;
+  std::memset(&transfer, 0, sizeof(transfer));
+
+  int transfer_completed = 0;
+  if (transfer_type == LIBUSB_TRANSFER_TYPE_BULK) {
+    libusb_fill_bulk_transfer(&transfer, device_handle, endpoint_address, data,
+                              length,
+                              /*callback=*/
```

Comment:
nit: The formatting here looks weird to me - why is there a newline here?

Reply:
Done - Was probably the case where clang-tidy fails on the side of respecting original newlines (thinking that they're intentional, meanwhile they're not).